README.md

```diff
@@ -1,23 +1,33 @@
-# operator-template
+# PgBouncer Kubernetes Operator
 
 ## Description
 
-TODO: Describe your charm in a few paragraphs of Markdown
+The PgBouncer Kubernetes Operator deploys and operates the [PgBouncer](https://www.pgbouncer.org) lightweight connection pooler for PostgreSQL.
 
 ## Usage
 
-TODO: Provide high-level usage, such as required config or relations
+As this charm is not yet published, you need to follow the build and deploy instructions from [CONTRIBUTING.md](https://github.com/canonical/pgbouncer-k8s-operator/CONTRIBUTING.md).
 
 ## Relations
 
-TODO: Provide any relations which are provided or required by your charm
+#### Planned
+- `db:[pgsql](https://github.com/canonical/ops-lib-pgsql/)`
+- `db-admin:[pgsql](https://github.com/canonical/ops-lib-pgsql/)`
+- `backend-db-admin:[pgsql](https://github.com/canonical/ops-lib-pgsql/)`
+  - Provides a relaton to the corresponding [postgresql-k8s-operator charm](https://github.com/canonical/postgresql-k8s-operator).
+
+The following relations provide support for the [LMA charm bundle](https://juju.is/docs/lma2), our expected observability stack.
+
+- `prometheus:prometheus_scrape`
+- `loki:loki_push_api`
+- `grafana:grafana_dashboards`
 
 ## OCI Images
 
-TODO: Include a link to the default image your charm uses
+This charm uses the canonical pgbouncer-container docker image, available here: [https://github.com/canonical/pgbouncer-container](https://github.com/canonical/pgbouncer-container). As this container has not been uploaded anywhere, **you will need to build this image locally before use**, following the instructions in [CONTRIBUTING.md]([CONTRIBUTING.md](https://github.com/canonical/pgbouncer-k8s-operator/CONTRIBUTING.md)).
 
```

Comment:
Similar thing as the CONTRIBUTING one.

```suggestion

## License
The Charmed MongoDB Operator is free software, distributed under the Apache Software License, version 2.0. See [LICENSE](https://github.com/canonical/mongodb-operator/blob/main/LICENSE) for more information.


## Security
Security issues in the Charmed MongoDB Operator can be reported through [LaunchPad](https://wiki.ubuntu.com/DebuggingSecurity#How%20to%20File). Please do not file GitHub issues about security issues.

```

Reply:
Thanks for catching this, I've added it in (with links to this repo, instead of mongodb ;) )